src/translations/maps/indexMap.js

```diff
@@ -0,0 +1,44 @@
+import BaseMap from './baseMap';
+import { getListWithRemovedItems, getListWithInsertedItems } from './utils/visuallyIndexed';
+import { getDecreasedIndexes, getIncreasedIndexes } from './utils/actionsOnIndexes';
+
+/**
+ * Map for storing mappings from an visual index to an physical index.
+ */
+class IndexMap extends BaseMap {
+  constructor() {
+    // Not handling custom init function or init value.
+    super(index => index);
+  }
+
+  /**
+   * Add values to list and reorganize.
+   *
+   * @private
+   * @param {Number} insertionIndex Position inside actual list.
+   * @param {Array} insertedIndexes List of inserted indexes.
```

Comment:
I believe it should be:

```
   * @param {Number} insertionIndex Position inside the list.
   * @param {Array} insertedValues Array of inserted values.
```

`BaseMap` uses the word `index` only to refer to the keys in `this.list` and the word `value` to refer to the values.

It is not clear if you using the words "list" and "array" as synonyms. I'd rather say "array" when you mean just an instance of JS array, and "list" if you mean the special data structure used by `BaseMap`.

Reply:
The first parameter changed: https://github.com/handsontable/handsontable/pull/5945/commits/6e21a3a50ba2114035f8b236d90d7e523f096168#diff-63f22dac0051b851c24a76fb40419c1cR18

Reply:
Why not the second parameter? Line 15 says `Add values to list and reorganize.` and line 19 says `List of inserted indexes.`. 

I am confused that the same word `index` is used:
- on line 18 to mean `key` 
- on line 19 to mean `value`

I think that it is better to avoid using the word "index" at all rather than using it in ambigous ways.

Since we don't have strict typing, accurate property names and descriptions are our only way to improve code cognition.